File: src/main/java/com/baedal/store/domain/model/Store.java
package com.baedal.store.domain.model;

import java.time.LocalDateTime;
import lombok.Builder;
import lombok.Getter;

@Builder
@Getter
public class Store {

  private String name;

  private String title;

  private String content;

  private String address;

  private String pictureUrl;

  private String category;

  private LocalDateTime openTime;

  private LocalDateTime closeTime;
}

File: src/main/java/com/baedal/store/kotlin/domain/model/Store.kt
package com.baedal.store.kotlin.domain.model

import java.time.LocalTime

data class Store(

    val id: Long,

    val ownerId: Long,

    val name: String,

    val title: String,

    val content: String,

    val address: String,

    val pictureUrl: String,

    val category: String,

    val openTime: LocalTime,

    val closeTime: LocalTime,

    val deliveryAmount: Int
)
